examples/passing/EmptyTypeClass.purs

```diff
@@ -1,12 +1,11 @@
 module Main where
 
 import Prelude
+import Control.Monad.Eff
+import Control.Monad.Eff.Console
 
-class PartialP
-
-head :: forall a. (PartialP) => Array a -> a
+head :: forall a. Partial => Array a -> a
 head [x] = x
 
-instance allowPartials :: PartialP
-
-main = Control.Monad.Eff.Console.log $ head ["Done"]
+main :: Eff _ _
+main = log "Done"
```

Comment:
Possibly, I think you're probably better equipped than I am to decide whether it's worth doing.